cmd/stanza/root.go

```diff
@@ -87,16 +75,20 @@ func NewRootCmd() *cobra.Command {
 }
 
 func runRoot(command *cobra.Command, _ []string, flags *RootFlags) {
-	logger := newLogger(*flags).Sugar()
+	conf, err := service.LoadConfig(flags.PluginDir, flags.ConfigFile)
+	if err != nil {
+		log.Fatalf("Failed to load config: %s", err)
+	}
+
+	logger := service.NewLogger(*conf.Logging).Sugar()
 	defer func() {
 		_ = logger.Sync()
```

Comment:
Ok, I think I see why this was done. `make lint` complains on this line if we don't explicitly ignore the error like this.

I notice now that there's actually a lot of linting errors from the stuff we've done recently; I guess we don't have any CI check for it, which seems bizarre.

We should follow up on those linter errors in a follow up PR (probably after CI stuff is cemented). In this specific case, we might want to just leave it as is, since it's done purposefully.

Alternative is to have one of those ignore comments.

Reply:
Ok we can skip linter stuff. I'll create a ticket to setup [revive](https://github.com/mgechev/revive), the linter we're switching to, and fix all lint errors that generates.